bedrock/foundation/templates/foundation/annualreport/2009/broadening-our-scope.html

```diff
@@ -183,11 +183,11 @@ <h4 class="section-title" id="drumbeat">
       <p>
         {% trans
           logo=static('img/foundation/annualreport/2009/drumbeat-logo.jpg'),
-          webcraft="http://www.drumbeat.org/",
-          festival="http://www.drumbeat.org/",
-          movies="http://www.drumbeat.org/",
-          subtitles="http://www.drumbeat.org/",
-          fund="https://donate.mozilla.org/"
+          webcraft="//www.drumbeat.org/webcraft",
```

Comment:
It looks like you've accidentally reverted the changes when rebasing. This needs fixing again.